defold-spine/src/gui_node_spine.cpp

```diff
@@ -307,7 +309,8 @@ bool SetScene(dmGui::HScene scene, dmGui::HNode hnode, dmhash_t spine_scene)
     node->m_HasNextCallback = 1;
     node->m_NextCallback = 0;
 
-    return SetupNode(spine_scene, resource, node, true);
+    bool create_bones = node->m_FindBones ? false : true; // See comment in GuiClone()
+    return SetupNode(spine_scene, resource, node, create_bones);
```

Comment:
In short, we'll find the gui nodes the next frame.
The cloning of a node is dealt with by the gui system, and it clones the node and its bones.